src/test/createHierarchyRenderer.js

```diff
@@ -47,7 +47,7 @@ var React = require('React');
  * each invocation of the hierarchy renderer. If new classes are needed, you
  * should re-execute `createHierarchyRenderer` with the same arguments.
  *
- * @param {array<function>} ...renderMethods
+ * @param {array<function>} renderMethods
```

Comment:
I guess the param is still called `renderMethods` even if it has a spread operator